xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
Sorry if we already discussed - should we try to implicitly rely on Rouge like we did on Pygments?
That way we're still zero dependencies and will colorize if someone has installed Rouge